bank_transaction.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+
+int main(void)
+{
+int current_balance,transfer_amount;
+int *ptr_current_balance, *ptr_transfer_amount;
+int sum, substact;
+int *new_balance,*new_state;
```

Comment:
Our coding standard requires that each variable be declared on its own line.
also spelling on subtract?

```suggestion
int current_balance;
int transfer_amount;
int *ptr_current_balance
int *ptr_transfer_amount;
int sum
int subtract;
int *new_balance
int *new_state;
```